test/unit/test_registry_get_request_exception.py

```diff
@@ -6,4 +6,4 @@
 class testRegistryGetRequestException(unittest.TestCase):
     def test_registry_request_error_catching(self):
         # using non routable IP to test connection error logic in the _get function
-        self.assertRaises(RegistryException, _get, '', 'http://10.255.255.1')
+        self.assertRaises(RegistryException, _get, '', 'http://0.0.0.0')
```

Comment:
A little bonus update-- this should make this test a lot faster (0.0.0.0 doesn't even get as far as the DNS level before it fails routing)